perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -1910,13 +1912,22 @@ public boolean setAttributeWithoutCheck(PerunSession sess, Facility facility, At
 	}
 
 	@Override
-	public void setAttribute(PerunSession sess, Vo vo, Attribute attribute) throws WrongAttributeValueException, WrongAttributeAssignmentException, WrongReferenceAttributeValueException {
+	public void setAttribute(PerunSession sess, Vo vo, Attribute attribute) throws WrongAttributeValueException, WrongAttributeAssignmentException, WrongReferenceAttributeValueException, InvalidHtmlInputException {
 		convertEmptyAttrValueToNull(attribute);
 		if (attribute.getValue() == null) {
 			removeAttribute(sess, vo, attribute);
 			return;
 		}
 		checkAttributeSyntax(sess, vo, attribute);
+		if (attribute.getFriendlyName().equals("htmlMailFooter")){
```

Comment:
this is not the right place to check attribute value - this is usually handled in separate attribute module (checkAttributeSyntax method of VoAttributesModuleImplApi inheriting classes)